api/routes/tankroutes.js

```diff
@@ -11,6 +11,14 @@ const Tank = require("../models/Reading");
  * @returns {Object} Tank object 
  * @throws {NotFoundError} When tank is not created.
  */
+
+/**
+ * @swagger
+ * tags:
+ *   name: Tanks
```

Comment:
```suggestion
 *   name: Add a new tank
```

Reply:
This is the tag this has nothing to do with the documentation,Tank route was not in the server.js file so I haven't made documentation for tank routes.

Tag is the section under which its following routes come under

Reply:
Oh yes. Don't change the tag